### DISCUSSION TITLE
Overriding Input components placeholder font-style

### DISCUSSION THREAD

**Discussion by jared-hunter** [twilio-labs/paste#3185] [Q&A] [answered]
As can be seen in the screenshot, i’m trying to target the ::placeholder so i can override the font-style from italic to normal
i’ve tried using ‘styled’ component to target it along with many variations of targeting the element name  but i cant seem to find anyway to get it to modify it

Does anyone have any guidance on how i can change that style or perhaps its just not possible?

my styled component attempt:
```javascript
const StyledInput = styled(Input)`
& ::placeholder {
    font-style: normal;
    opacity: 0.42;
  }
}`;
```

my customizations using the customization provider
```javascript
{
  WORKER_TRANSFER_DIR_SEARCH_BOX: {
    visibility: 'visible',
  },
  WORKER_TRANSFER_DIR_SEARCH_BOX_PREFIX: {
    padding: '0.4379rem 0.5rem',
  },
  WORKER_TRANSFER_DIR_SEARCH_BOX_ELEMENT: {
    fontWeight: '400',
  },
}
```
my react component:
```javascript
<Input
  element="WORKER_TRANSFER_DIR_SEARCH_BOX"
  insertBefore={ <SearchBox/> }
  type="text"
  key="custom-directory-input-field"
  placeholder="Search"
/>
```

<img width="1216" alt="Screenshot 2023-04-21 at 09 42 42" src="https://user-images.githubusercontent.com/44647761/233670402-4a85ae85-4708-4e3f-8e79-761586273e68.png">

**Comment by SiTaggart** [comment, marked as answer]
Hi @jhunter-twilio 

You are so close! You can do everything you need via the elements object on the Customization Provider. It will accept css selectors as keys to each Element you target. And obviously you have to target the input HTML element itself.

So you would write something like 

```
WORKER_TRANSFER_DIR_SEARCH_BOX_ELEMENT: {
  "&::placeholder": {
    fontStyle: "normal"
  }
}
```

At the end of the day, the object for `WORKER_TRANSFER_DIR_SEARCH_BOX_ELEMENT` is just a Emotion CSS style object. It can do everything Emotion can do with style objects, but we type some of the values to only be design tokens where necessary.

Here's a Code sandbox to show you what I mean to achieve overriding the placeholder styles on the Input. https://codesandbox.io/s/input-placeholder-style-override-b1lc4f
